pallets/cash/src/events.rs

```diff
@@ -119,7 +112,7 @@ mod tests {
         ];
 
         let fetch_from = blocks_to_return[0].number;
-        let fetch_to = blocks_to_return[blocks_to_return.len() - 1].number;
+        let fetch_to = blocks_to_return[blocks_to_return.len() - 1].number + 1;
```

Comment:
Or this is not: it's actually a closed interval now?

Reply:
Its half-open, we can change the log